### PR TITLE
update env file for v3.1 release

### PR DESCRIPTION
**Pull request by carolinemodic**
Update env file for v3.1.0 release

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -13,8 +13,6 @@ REACT_APP_VX_SKIP_BALLOT_PACKAGE_AUTHENTICATION=FALSE
 REACT_APP_VX_SKIP_CAST_VOTE_RECORDS_AUTHENTICATION=FALSE
 REACT_APP_VX_CAST_VOTE_RECORD_OPTIMIZATION_EXCLUDE_ORIGINAL_SNAPSHOTS=TRUE
```
Comment (carolinemodic) [inline .env:14]: @arsalansufi Do we still want these two CVR optimizations as true?

File: .env [hunk 2]
```diff
@@ -13,8 +13,6 @@ REACT_APP_VX_SKIP_BALLOT_PACKAGE_AUTHENTICATION=FALSE
...
 REACT_APP_VX_CAST_VOTE_RECORD_OPTIMIZATION_EXCLUDE_REDUNDANT_METADATA=TRUE
-REACT_APP_VX_DISABLE_BALLOT_BOX_CHECK=FALSE
-REACT_APP_VX_CONVERTER=ms-sems
-REACT_APP_VX_PRECINCT_REPORT_DESTINATION=thermal-sheet-printer
-REACT_APP_VX_DISABLE_BALLOT_BOX_CHECK=FALSE
+REACT_APP_VX_DISABLE_BALLOT_BOX_CHECK=TRUE
```
Comment (carolinemodic) [inline .env:16]: What is this? Switching out the ballot bag? @mattroe

**Review by benadida — APPROVED** [review]
Recommend getting arsalan's confirmation on those two bits, but otherwise looks good.

File: .env (hunk 1 above)
Comment (arsalansufi) [reply .env:14]: Yup, the two CVR optimizations should be kept enabled!

**Review by arsalansufi — APPROVED** [review]

**Pull request merged**